src/main/scala/com/gu/newsletterlistcleanse/services/Newsletters.scala

```diff
@@ -1,14 +1,61 @@
-package com.gu.newsletterlistcleanse
+package com.gu.newsletterlistcleanse.services
 
 import java.time.ZonedDateTime
 
+import cats.implicits._
+import cats.data.EitherT
 import com.gu.identity.model.{EmailNewsletter, EmailNewsletters}
 import com.gu.newsletterlistcleanse.db.ActiveListLength.getActiveListLength
 import com.gu.newsletterlistcleanse.db.{ActiveListLength, CampaignSentDate}
 import com.gu.newsletterlistcleanse.models.NewsletterCutOff
+import io.circe.Decoder
+import sttp.client.asynchttpclient.WebSocketHandler
+import sttp.client.{SttpBackend, basicRequest}
+
+import scala.concurrent.duration._
+import io.circe.parser.decode
+import io.circe.generic.semiauto._
+import sttp.client._
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.Future
+import scala.concurrent.duration.FiniteDuration
+
+case class Newsletter(brazeNewsletterName: String)
+
+object Newsletter {
+  implicit val newsletterDecoder: Decoder[Newsletter] = deriveDecoder[Newsletter]
+}
 
 class Newsletters {
-  def allNewsletters: List[String] = EmailNewsletters.allNewsletters.map(_.brazeNewsletterName)
+  val timeout: FiniteDuration = 5000.seconds
```

Comment:
Should this be either `5000.millis` or `5.seconds`?
```suggestion
  val timeout: FiniteDuration = 5.seconds
```